app/errors.py

```
@@ -0,0 +1,17 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    def __init__(self, message: str = "NotVaccinatedError") -> None:
+        super().__init__(message)
+
+
+class OutdatedVaccineError(VaccineError):
+    def __init__(self, message: str = "OutdatedVaccineError") -> None:
+        super().__init__(message)
+
+
+class NotWearingMaskError(Exception):
+    def __init__(self, message: str = "NotWearingMaskError") -> None:
```

This violates checklist item #3: 'Use descriptive error messages'. Consider a more helpful default message, such as "Visitor is not wearing a mask."